Makefile

```diff
@@ -19,13 +19,29 @@ test:
 single-docs:
 	sphinx-build docs/source docs/_build -a
 
+
+# update requirements.txt
+lock:
+	cd src; poetry lock
+	cd src; poetry export -f requirements.txt --without-hashes --output requirements.txt
+
 # Build the docker
-build:
+build-app:
 	docker build . -t pyronear/pyro-engine:python3.8.1-slim
```

Comment:
why do you build two images here ?

Reply:
hum build-app is building one image, with the app contained in src/ and build-lib is building the library locally.
Maybe build-image and build-lib would be a better wording ?

Reply:
Sorry, maybe it's just me, but I don't understand why we need these two lines, we can't just keep :

`docker build . -t pyronear/pyro-engine:latest`

Reply:
done @MateoLostanlen